scalability_and_performance/low_latency_tuning/cnf-debugging-low-latency-tuning-status.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-debugging-low-latency-tuning-status"]
+= Debugging low latency node tuning status
+include::_attributes/common-attributes.adoc[]
+:context: cnf-debugging-low-latency
+
+toc::[]
+
+Use the `PerformanceProfile` custom resource (CR) status fields for reporting tuning status and debugging latency issues in the cluster node.
+
+include::modules/cnf-debugging-low-latency-cnf-tuning-status.adoc[leveloffset=+1]
```

Comment:
cnf-debugging-low-latency-cnf-tuning-status-machineconfigpool is burried in a module but should in be listed in this assembly. Modules within modules might cause issues, but maybe I'm misunderstanding something here?

Reply:
Will rework separately under https://issues.redhat.com/browse/TELCODOCS-1733